ecosystem/platform/server/app/views/nft_offers/show.html.erb

```diff
@@ -44,7 +44,7 @@
                 <% if step.name == :sign_in %>
                   To begin, sign in to Aptos.
                 <% elsif step.name == :connect_wallet %>
-                  Connect your wallet.
+                  Connect your wallet. You will need an <%= render(LinkComponent.new(href: 'https://chrome.google.com/webstore/detail/petra-aptos-wallet/ejjladinnckdgjemekebdpeokbikhfci')) { 'Aptos Wallet' } %> with a funded account and the network set to <%= @nft_offer.network.capitalize %>.
```

Comment:
nit: `Petra Wallet`

Reply:
Or maybe you are preparing for other wallets to integrate

Reply:
Yeah I want to keep it generic. Joe mentioned that we could have a modal that pops up with a grid of all the supported wallets.

Reply:
Updated copy to keep it generic but mention Petra specifically, per Max's suggestion.